starlark/unpack.go

```diff
@@ -173,14 +188,19 @@ func UnpackPositionalArgs(fnname string, args Tuple, kwargs []Tuple, min int, va
 		return fmt.Errorf("%s: got %d arguments, want %s%d", fnname, len(args), atmost, max)
 	}
 	for i, arg := range args {
-		if err := unpackOneArg(arg, vars[i]); err != nil {
+		if err := unpackOneArg(arg, vars[i], ""); err != nil {
 			return fmt.Errorf("%s: for parameter %d: %s", fnname, i+1, err)
 		}
 	}
 	return nil
 }
 
-func unpackOneArg(v Value, ptr interface{}) error {
+func unpackOneArg(v Value, ptr interface{}, paramSpec string) error {
```

Comment:
Let's pass `skipNone bool` here and move the check to the caller. This function has no business with the name.

Reply:
done